v5_MIGRATION_GUIDE.md

```diff
@@ -35,17 +35,18 @@ val cardClient = CardClient(context, "TOKENIZATION_KEY_OR_CLIENT_TOKEN")
 
 The `paypal-data-collector` module has been removed and replaced by the `data-collector` module.
 The `DataCollector` class within the `data-collector` module has the same
-`collectDeviceData` methods, so if you were previously using the `paypal-data-collector` library,
-no code changes are required aside from updating your dependency.
+`collectDeviceData` methods.
 
 If you were using the `data-collector` library in v4, `DataCollector#collectDeviceData(context,
 merchantId, callback)` is now `DataCollector#collectDeviceData(context, riskCorrelationId,
 callback)`, where `riskCorrelationId` is an optional client metadata ID.
 
 ```kotlin
 val dataCollector = DataCollector(context, authorization)
-dataCollector.collectDeviceData(context) { deviceData, error ->
-    // send deviceData to your server
+dataCollector.collectDeviceData(context) { result ->
+    if (result is DataCollectorResult.Success) {
+        // send result.deviceData to your server
+    } 
```

Comment:
Do we want to show handling the error result here?

Reply:
I didn't include it because technically they can still proceed without the device data, so on error, just don't send the device data

Reply:
Got it, that makes sense